### ISSUE TITLE
About the loss of training

### ISSUE THREAD

**Issue by tangchen2** [YangZeyu95/unofficial-implement-of-openpose#46]
Exceuse me, I have trained the model according to your steps, i find that the model only trained 5 epochs and the final loss is around 215 ~ 230, and i have test the ski image, the performance is not much accurate but not too bad. Can i improve the model performance by training more epochs and what about the learning rate set? Could you give me some advice, thank you so much.

**Comment by YangZeyu95** [comment]
More epoch is necessary. My lr init as 1e-4 with 0.9 decay every epoch

**Comment by tangchen2** [comment]
Thank you for your reply! I will try more epochs with your lr set
